### PR TITLE
Add -i to exec for compatibility reasons

### PR DESCRIPTION
**Pull request by baude**
No change to functionality

Resolves: #606

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
@baude Can you remove the "Resolves:"? We really ought to implement `exec -i` properly at some point

**Comment by mheon** [comment]
Otherwise LGTM

**Comment by baude** [comment]
well it does resolve the issue ... what word would you prefer?

**Comment by mheon** [comment]
@baude I don't feel like it does, given that properly resolving the issue would be implementing `-i`. Maybe we should make a new issue for that, though...

File: cmd/podman/exec.go [hunk 1]
```diff
@@ -19,6 +19,10 @@ var (
 			Name:  "privileged",
 			Usage: "Give the process extended Linux capabilities inside the container.  The default is false",
 		},
+		cli.BoolFlag{
+			Name:  "interactive, i",
+			Usage: "Keep STDIN open even if not attached",
```
Comment (rhatdan) [inline cmd/podman/exec.go]: Lets add a note that this actually does not do anything like it is in the man page.

Comment (baude) [reply cmd/podman/exec.go]: fixed

**Comment by rhatdan** [comment]
LGTM

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

File: cmd/podman/exec.go [hunk 2]
```diff
@@ -19,6 +19,10 @@ var (
 			Name:  "privileged",
 			Usage: "Give the process extended Linux capabilities inside the container.  The default is false",
 		},
+		cli.BoolFlag{
+			Name:  "interactive, i",
+			Usage: "Not supported.  All exec commands are interactive by default.",
```
Comment (TomSweeneyRedHat) [inline cmd/podman/exec.go]: nit, I know I love periods and would prefer to have them in usage statements, but our other usage statements don't end with them.

**Comment by mheon** [comment]
Logs test flaked
@rh-atomic-bot retry

**Pull request closed without merging**